test/utilities.py

```diff
@@ -18,6 +19,62 @@
     )
 
 
+class TempFile:
+    """docstring for TempFile"""
+
+    def __init__(self, name, handle):
+        super(TempFile, self).__init__()
+        self.name = name
+        self.handle = handle
+
+    def read_whole_file(self):
+        while True:
+            if not self.handle.read(10):
```

Comment:
Why reading by chunks of 10?

Reply:
changed it to 10k now, just debugging

Reply:
Why not simply omitting this parameter ?
_cf._ https://docs.python.org/3/tutorial/inputoutput.html#methods-of-file-objects